OrcaStars/app/build.gradle

```diff
@@ -8,8 +8,8 @@ android {
         applicationId "edu.cascadia.mobas.orcastars"
         minSdkVersion 21
         targetSdkVersion 28
-        versionCode 1
-        versionName "0.1"
+        versionCode 2
+        versionName "0.2"
```

Comment:
I'm trying to make the releases more self-descriptive. 
Please change the versionName to "Closed Test: Winter 2020"